seeders/20191112083122-demo-users.js

```diff
@@ -4,9 +4,9 @@ const passwordHash = hash.generateSync('12345678');
 
 module.exports = {
   up: (queryInterface, Sequelize) => queryInterface.bulkInsert('users', [{
-    firstName: 'John',
-    lastName: 'Doe',
-    email: 'john@barefoot.com',
+    firstName: 'Super',
+    lastName: 'Administrator',
+    email: 'super@administrator.com',
     password: passwordHash,
     createdAt: Sequelize.literal('CURRENT_TIMESTAMP'),
     updatedAt: Sequelize.literal('CURRENT_TIMESTAMP'),
```

Comment:
You need to add a field `role: 'super_administrator'` because it is not possible now to change the user role.

Reply:
Thanks for picking up on this Shaphan.

Reply:
And also set admin as verified